DfE.FindInformationAcademiesTrusts.Data.AcademiesDb/TrustSearch.cs

```diff
@@ -10,12 +9,6 @@ public class TrustSearch : ITrustSearch
     private readonly IAcademiesDbContext _academiesDbContext;
     private const int PageSize = 20;
 
-    [ExcludeFromCodeCoverage] // This constructor is used by the DI container and is not unit testable
```

Comment:
I can see this has been removed in a couple places, is this being added as an exclusion on SonarCloud? is that why it's removed?

Reply:
The first commit on the branch explains this one - essentially this constructor (using the concrete `AcademiesDbContext` rather than the interface `AcademiesDbContext`) is no longer needed because we've set up the DI container better. This means we could remove this effectively duplicated code and not introduce the same overhead to the new repository types

Reply:
(it was annoying having to update the dependencies in to places)